### ISSUE TITLE
Request - agent access for operators

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8625]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

All agent access is via the agent API, which currently interprets a private flag on https://arctos.database.museum/info/ctDocumentation.cfm?table=ctagent_attribute_type as "for manage_agent operators only." Therefore, agent picks for operators without that access look like...

![Image](https://github.com/user-attachments/assets/ab02976a-8627-4560-83e7-98ad7c125483)

... this. Picking agents is probably a less-precise exercise than it should be for these operators, and picking addresses for transactions isn't possible at all.

I think the API should be adjusted to return all information for all coldfusion_user holders, and I'll plan to move this to dev and make that adjustment in a week or two if there are no objections.

Ping @ccicero

**Comment by mkoo** [comment]
makes sense

**Comment by dustymc** [comment]
enthusiastically supportive silence accepted, going dev
